torch/_inductor/kernel/mm_scaled.py

```diff
@@ -551,8 +551,9 @@ def tuned_scaled_mm(
                 )
         else:
             for config in scaled_mm_configs(m, n, k):
-                if k == 16 and config.kwargs["BLOCK_M"] >= 64:
-                    continue  # Triton crashes in this case
+                # K dim must be >= 32 for tcgen05.mma.kind::f8f6f4.* PTX instruction to be valid
+                # source: https://docs.nvidia.com/cuda/parallel-thread-execution/#tcgen05-matrix-shape
+                if k < 32: continue
```

Comment:
Lets do a arch check so that we only add this if we are on b200, 

you can create a helper like this;
https://github.com/pytorch/pytorch/blob/1cb4e2df659059ac85cfe6f64a2363a12ae1071a/torch/_inductor/utils.py#L1219-L1221

Reply:
Sure, added a check here